unittests/scans/rusty_hog/gottingenhog_one_vuln.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "stringsFound": [
+      "password: jeans"
+    ],
+    "issue_id": "TEST-123",
```

Comment:
issue_id referse to the JIRA Ticket. However, a JIRA Ticket can have multiple findings. Thus, this is not a good candidate anymore as it will not be unique in these cases.